tools/tiled_ensemble/ensemble_tiler.py

```diff
@@ -0,0 +1,90 @@
+"""Tiler used with ensemble of models."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from omegaconf import DictConfig, ListConfig
+from torch import Tensor
+
+from anomalib.pre_processing.tiler import Tiler, compute_new_image_size
+
+
+class EnsembleTiler(Tiler):
```

Comment:
I like the design of this class in combination with the `TileCollater`. It's clean and easy to follow. Good job :)